src/System.Numerics.Vectors/src/System/Numerics/Vector2_Intrinsics.cs

```diff
@@ -50,7 +50,7 @@ public Vector2(float x, float y)
         /// <param name="array">The destination array.</param>
         [Intrinsic]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public readonly void CopyTo(float[] array)
+        public void CopyTo(float[] array)
```

Comment:
Why did you remove the `readonly`?

Reply:
ah crud. That was a mistake. I should have been more careful. The only change was the addition of the function (one to each file).

Reply:
I must've been working from an old version

Reply:
Could you fixup the diff? Once that is done, I think it looks basically complete modulo tests :)